index.html

```diff
@@ -10,12 +10,48 @@
     <link href="main.css" rel="stylesheet" type="text/css">
   </head>
   <body>
+
+    <!-- Facebook plugin related -->
+    <div id="fb-root"></div>
+    <script async defer crossorigin="anonymous" src="https://connect.facebook.net/pt_PT/sdk.js#xfbml=1&version=v3.3"></script>
+
     <header>
       Geeksessions [LOGO]
     </header>
     <main>
       <p>We organize regular events like talks and workshops with the goal of bringing local tech denizes together.</p>
     </main>
+    <div class="row space-around ">
+      <div>
+        <div class="row">
+          <p>
+            Join us on Discord: &nbsp
+          </p>
+          <a href="https://l.facebook.com/l.php?u=https%3A%2F%2Fdiscord.gg%2FtJmtAUS%3Ffbclid%3DIwAR0tcQMX8dGnq-O9EPYQpkYc1ueQO357pToJ9KhPj0f05M1h7T9MVUbFzhU&h=AT0K0RNhetLCr2ZY5mTlChOfZtWicKrtDfJhX3xPCvYMLzQLTH3VoMuQi4sMNxv5fWm0fHMyleorZmDzb2FiHejgRmyDQDM7G91WfqaeZYCJDdiiZwNIdDS_ZCWA_OPk16Hda8VEErXE8RqcNUIeLUWPMAmc2TFOate9VIoL-qkAefypMPHtbe_rMrBWzp8vbeXHffsfYbsNfdkp09liUcvV9dxgPPJaEZYbm-vcCXkohLv7I3CAmt3fHgCI6bNz8nvMm5jvCAZTYTzekEWGJ2Lu5ZsabnEWoJaPc6TcNzASgvGWlmr4AQcaMftDkPxcw6dlF7ov7aUlCXYx1TA3pHyjWdgzY74NgDeNJfLZLYkOb_tBD44clkeOP0gyHWNR-CNMRmRNWyKr3ThysxjSf4KLInm62EvNS1LgZRqHzhBBFvDzEXToyn9xTFvZUbaFeDzJ3UsFLxVB4heNmUCE5FsgLz7oTOEXaDYmY_vzRH314riqvoNME4RE3WtN">
```

Comment:
Let's change the Discord link to point to https://chat.geeksessions.io 💪

Reply:
Fixed 👍